### PR TITLE
Add ESCODEX

### PR DESCRIPTION
**Pull request by kulibih**
<h2>General</h2>
Closes #[Insert issue number]

[Insert description of what you did, any description how to test or check your changes are welcome]

<h2>General</h2>
lease make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at 
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [ ] Check for unused code
- [ ] No unrelated changes are included
- [ ] None of the changed files are reformatting only
- [ ] Code is self explanatory or documented
- [ ] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [ ] Chrome 
- [ ] Opera
- [ ] Firefox
- [ ] Safari

<h2>User interface changes</h2>

_Delete this section if there weren't any UI changes. Please make sure you tested your changes in all themes_

- [ ] Dark
- [ ] Light
- [ ] Midnight

_Please provide screenshots/licecap of your changes below_

### CODE REVIEW

File: app/branding.js [hunk 1]
```diff
@@ -307,7 +309,7 @@ export function getAssetNamespaces() {
  */
 export function getAssetHideNamespaces() {
     // e..g "OPEN.", "bit"
-    return [];
+    return ["ESCODEX."];
```
Comment (startailcoon) [inline app/branding.js]: This is not an acceptable change.

**Comment by startailcoon** [comment]
@kulibih can you please post some details about what ESCODEX are.

**Comment by sschiessl-bcp** [comment]
In order to be listed you need to fulfilll basic gateway requirements. Please contact me via telegram

**Comment by kulibih** [comment]
some details about what ESCODEX
http://www.escodex.com/
https://wallet.escodex.com/
charlie@escodex.com
https://discord.gg/Wpa7EGq

File: app/branding.js (hunk 1 above)
Comment (kulibih) [reply app/branding.js]: I agree and I made this change after applying for mergers.
It shouldn't have come here.

**Comment by sschiessl-bcp** [comment]
I see you closed this, I assume you are not interested anymore to be listed?

**Comment by kulibih** [comment]
I do not know how to make such requests correctly and there we don’t need
the data that was entered by me after the request was submitted. I think
that someone will do it better than me.
there is a problem Marked it out please understand you

пт, 3 мая 2019 г. в 16:29, Stefan Schießl <notifications@github.com>:

**Pull request closed without merging**